ntbs-service/Pages/Shared/_NotificationDetails.cshtml

```diff
@@ -0,0 +1,84 @@
+@using ntbs_service.Models.Enums;
+@{
+    var NotificationStatusCssClass = "";
+    switch(@Model.NotificationStatus)
+        {
+            case NotificationStatus.Draft: NotificationStatusCssClass = "notification-banner--draft";
+                break;
+            case NotificationStatus.Notified: NotificationStatusCssClass = "notification-banner--notified";
+                break;
+            case NotificationStatus.Denotified: NotificationStatusCssClass = "notification-banner--denotified";
+                break;
+            default:
+                break;
+        }
+    var isBanner = ViewData.ContainsKey("IsBanner") && (bool)ViewData["IsBanner"];
+
+    var RedirectPath = @Model.NotificationStatus == NotificationStatus.Draft ? @Model.PatientEditPath : @Model.OverviewPath;
+}
+
+<dl class="notification-banner @NotificationStatusCssClass">
+    <div class="notification-banner-head">
+        <nhs-grid-row classes="notification-overview-details">
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <h1 class="notification-banner-title-text">
+                    <div> @Model.NotificationStatusString </div>
+                    @if(isBanner)
+                    {
+                        <div> #@Model.NotificationId </div>
+                    }
+                    else
+                    {
+                        <a href="@RedirectPath"> #@Model.NotificationId </a>

```

Comment:
We want the ids to be clickable and redirect to the linked notifications, but on the banner itself there is no link, hence this new logic